Cargo.toml

```diff
@@ -3,3 +3,11 @@
 
 [workspace]
 members = ["rust"]
+
+[profile.release]
+panic = "abort"
```

Comment:
Does this mean we don't get backtraces if something panics?

I would rather err on the side of having more debug output available.

Reply:
This will cause Synapse process to exit immediately on panic, I think for now we probably want to keep the current behaviour where (I believe) `pyo3` will catch the panic and turn that into an exception Python side.

Reply:
https://doc.rust-lang.org/cargo/reference/profiles.html#release

it's `unwind` by default ... I'll change that